this & object prototypes/foreword.md

```diff
@@ -1,21 +1,21 @@
 
-It wasn’t until 2005 that I first rediscovered JavaScript as a real programming language that I needed to pay closer attention to. After digging into the first beta release of Google Maps, I was hooked on the potential it had. At the time, Google Maps was a first-of-its-kind application -- it allowed you to move a map around with your mouse, zoom in and out, and make server requests without reloading the page -- all with JavaScript. It seemed like magic!
+Isso foi até 2005, quando eu tive minha primeira redescoberta do JavaScript como uma verdadeira linguagem de programação que eu precisava dar uma atenção especial. Depois de uma escavação na primeira versão beta do Google Maps, eu estava preso no potencial dele. Naquele momento, o Google Maps era uma aplicação das primeiras-do-seu-tipo -- ele permitiu que você movesse o mapa com seu mouse, dar zoom e retirar zoom, e fazer requisições ao servidor sem atualizar a página -- tudo isso com JavaScript. Isso parecia magia!
 
-When anything seems like magic, it is usually a good indication you are at the dawn of a new way of doing things. And boy, was I not wrong -- fast-forwarding to today, I would say that JavaScript is one of the primary languages I use for both client- and server-side programming, and I wouldn’t have it any other way.
+Quando alguma coisa parece magia, normalmente é um bom indicador de que você está no amanhecer de uma nova maneira de fazer as coisas. E garoto, eu não estava errado -- avanço-rápido para hoje em dia, eu devo dizer que JavaScript é uma das primeiras linguagens que eu usei tanto para a programação do lado do cliente quanto para no lado do servidor, e eu não teria outra forma de fazê-lo.
 
-One of my regrets as I look over the past 15 years is that I didn’t give JavaScript more of a chance before 2005, or more accurately, that I lacked the foresight to see JavaScript as a true programming language that is just as useful as C++, C#, Java, and many others.
+Um dos meus arrependimentos, quando eu olho para 15 anos atrás, é que eu não dei mais changes ao JavaScript antes de 2005, ou mais exatamente, eu não tinha a clareza de ver o JavaScript como uma linguagem de programação de verdade que é tão útil como C++, C#, e várias outras.
 
-If I had this *You Don’t Know JS* series of books at the start of my career, my career history would look much different than it does today. And that is one of the things I love about this series: it explains JS at a level that builds your understanding as you go through the series, but in a fun and informative way.
+Se eu tivesse essa série de livros *You Don't Know JS* quando eu comecei minha carreira, minha história provavelmente seria muito diferente do que é hoje. E isso é uma das coisas que eu adoro nesta série: ela explica JS da forma que você contrói seu entendimento a medida que você prossegue pela série, mas de uma forma divertida e informativa.
```

Comment:
> Nota: a tradução foi correta, mas como não traduzimos o titulo, ficou estranho.

Se eu tivesse ~~essa série de livros~~ **a série** *You Don't Know JS*
...
contrói seu entendimento a medida **em** que você prossegue ~~pela série~~ **por ela**